### ISSUE TITLE
OpenCL support

### ISSUE THREAD

**Issue by pdonahue-ventana** [intel/yarpgen#92]
The help message claims that the tool supports OpenCL:
```
$ ./yarpgen -h

yarpgen version 1.2 (build commit c9a772eab58c55d1 on 2020:04:09)
usage: yarpgen
	-q                        Quiet mode
	-v, --version             Print yarpgen version
	-d, --out-dir=<out-dir>   Output directory
	-s, --seed=<seed>         Predefined seed (it is accepted in form of SSS or VV_SSS)
	-m, --bit-mode=<32/64>    Generated test's bit mode
	--std=<standard>          Generated test's language standard
				  Default: c++11
				  Possible variants are: c++03, c++11, c++14, c++17, c++98, c11, c99, opencl_1_0, opencl_1_1, opencl_1_2, opencl_2_0, opencl_2_1, opencl_2_2
```

But [gen_policy.cpp](https://github.com/intel/yarpgen/blob/master/src/gen_policy.cpp#L124) will refuse to run if any of the opencl* variants are specified:
```
    if (options->is_c())
        allow_static_members = false;
    else if (options->is_cxx())
        allow_static_members = true;
    else {
        std::cerr << "ERROR at " << __FILE__ << ":" << __LINE__ << ": can't detect language subset" << std::endl;
        exit(-1);
    }
```

If OpenCL is not actually supported then the help message should be changed.

**Comment by Vsevolod-Livinskij** [comment]
Sorry, code from the experimental branch somehow landed in the trunk. 
Fixed in #94.

If you need compiler fuzzer for OpenCL, you can try [experimental branch](https://github.com/Vsevolod-Livinskij/yarpgen/tree/opencl), [DeepSmith](http://homepages.inf.ed.ac.uk/hleather/publications/2018_deepfuzzing_issta.pdf), or [CLFuzz](https://dl.acm.org/doi/pdf/10.1145/3366428.3380768).

**Comment by pdonahue-ventana** [comment]
Thanks for the pointers.
